flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionFactoryTest.java

```diff
@@ -50,7 +51,7 @@ public void testForceConsumptionOnReleaseDisabled() {
 	private static void testForceConsumptionOnRelease(boolean forceConsumptionOnRelease) {
 		ResultPartitionFactory factory = new ResultPartitionFactory(
 			new ResultPartitionManager(),
-			new NoOpIOManager(),
+			new FileChannelManager(new String[] {EnvironmentInformation.getTemporaryFileDirectory()}),
```

Comment:
we should actually close it at the end to not leak temp folders or introduce `FileChannelManager` interface and provide testing noop implementation, also for `ResultPartitionBuilder`.